Package.swift

```diff
@@ -13,6 +13,6 @@ let package = Package(
           targets: ["MSAL"]),
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->